src/main/java/persistence/entity/EntityLoader.java

```diff
@@ -0,0 +1,5 @@
+package persistence.entity;
+
+public interface EntityLoader {
+  <T> T load(Long id);
```

Comment:
```suggestion
public interface EntityLoader<T> {
  T findById(Long id);
  List<T> findAll(Long id);
```

단일 entity 의 load (findById) 뿐만 아니라,
모든 entity 의 load (findAll) 까지 구현 부탁드립니다.

Reply:
네 진행했습니다!